docs/run-preinstalled-wda.md

```diff
@@ -0,0 +1,70 @@
+---
+title: Run Preinstalled WebDriverAgentRunner
+---
+
+XCUITest driver can launch preinstalled WebDriverAgent directly against a real device.
+It lets you start an Appium session without the `xcodebuild` command execution to improve the session start performance.
+
+`appium:webDriverAgentUrl` let you manage everything related to the WebDriverAgent, but this functionality let Appium to manage instead.
+
+## For Real Device
+
+### Capabilities
+
+- Required
+  - `appium:usePreinstalledWDA` to start a WebDriverAgent process without xcodebuild
+- Optional
+  - `appium:preInstalledWDABundleId` to customize the WebDriverAgent bundle id Appium launches
```

Comment:
Lets not use Appium, but XCUITest driver everywhere. Appium is just a server and how nothing to do with iOS automation itself.